app-portage/lto-rebuild/files/lto-rebuild

```diff
@@ -14,14 +14,14 @@ if [ "$#" -eq 0 ] || [ "$1" == "-h" ]; then
 	echo "command will fail.  You can use -l to manually intervene."
 	echo
 	echo "Options:"
-	echo -e "-h\tDisplay this help"
-	echo -e "-l\tOnly list the packages that would be rebuilt and their slots"
-	echo -e "-r\tRebuild the packages using emerge (will ask for confirmation)"
+	printf "-h\tDisplay this help\n"
```

Comment:
you need to use `printf --` if the format string starts with a dash

```
% printf "-h\tDisplay this help\n"
dash: 1: printf: Illegal option -h
% printf -- "-h\tDisplay this help\n"
-h      Display this help
```

Reply:
Good catch, fixed